shared/actions/router.js

```diff
@@ -1,42 +1,5 @@
 import * as Constants from '../constants/router'
 
-/*
```

Comment:
I understand now.

You aren't switchingTabs, you are just being more specific about which tab you want to change the routing state.

so these would work as before, unless you give it the extra context of the tab. SGTM :+1: